doc/tools/kconfig/functions.rst

```diff
@@ -0,0 +1,53 @@
+.. _kconfig-functions:
+
+Kconfig Functions
+#################
+
+Kconfiglib provides user-defined preprocessor functions that
+we use in Zephyr to expose Device Tree information to Kconfig.
+For example, we can get the default value for a Kconfig symbol
+from the device tree.
+
+Device Tree Related Functions
+*****************************
+
+dt_int_val(kconf, _, name, unit):
+    This function looks up 'name' in the DTS generated "conf" style database
+    and if its found it will return the value as an decimal integer.  The
+    function will scale the value based on 'unit':
```

Comment:
Could you use either the word multiply or divide instead of scale, I'm not able to tell in which direction it scales the value.

Reply:
Changed to divide or division.